lib/jenkins_pipeline_builder/remote_dependencies.rb

```diff
@@ -28,7 +28,7 @@ def load(dependencies)
           file = entries[url]
         else
           opts = {}
-          opts = { ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE } if source[:verify_ssl] == false
+          opts = { ssl_verify_mode: OpenSSL::SSL::VERIFY_NONE } unless source[:verify_ssl]
```

Comment:
This now defaults to setting verify none when verify_ssl isn't passed. It needs to go back to the if verify_ssl == false.